README.md

```diff
@@ -15,18 +15,23 @@ This Rocket supports the following Providers:
 Add your rocket to your application in the configuration file.
 
 ### Generic
+
 ```typescript
+import { AllowedHttpMethods } from '@boostercloud/rocket-webhook-types'
```

Comment:
```suggestion
import { AllowedHttpMethod } from '@boostercloud/rocket-webhook-types'
```

Reply:
Fixed in commit a7212c64a7e659972a749aa9ada924be3d1d85b5.